pkg/controller/persistentvolume/persistentvolume_provisioner_controller.go

```diff
@@ -345,6 +347,11 @@ func newProvisioner(plugin volume.ProvisionableVolumePlugin, claim *api.Persiste
 		AccessModes:                   claim.Spec.AccessModes,
 		PersistentVolumeReclaimPolicy: api.PersistentVolumeReclaimDelete,
 		CloudTags:                     &tags,
+		ClusterName:                   controller.clusterName,
+	}
+
+	if pv != nil {
+		volumeOptions.PVName = pv.Name
```

Comment:
Makes sense, so we want/expect the name of the PV in the provider to be different then (contain more info then) the name of the API PV object.